src/python_optimiser_interface.cpp

```diff
@@ -871,6 +871,70 @@ extern "C"
     return PyBool_FromLong(optimiser->refine_partition);
   }
 
+  PyObject* _Optimiser_set_max_comm_size(PyObject *self, PyObject *args, PyObject *keywds)
+  {
+    PyObject* py_optimiser = NULL;
+    unsigned int max_comm_size = 0;
```

Comment:
The package actually uses `size_t` to represent unsigned integers. This ensures it is correctly compiled for 64-bit support (`int` is 32 bit), this is also how `max_comm_size` is defined in the actual C optimiser object.